.codecov.yml

```diff
@@ -11,234 +11,230 @@ coverage:
-      Ping:
+      pihole:
+        target: 75
+        flags:
+        - pihole
+      ping:
         target: 75
         flags:
         - ping
-      Portworx:
+      portworx:
         target: 75
         flags:
         - portworx
-      Puma:
+      puma:
         target: 75
         flags:
         - puma
-      PureFA:
+      purefa:
         target: 75
         flags:
         - purefa
-      PureFB:
+      purefb:
         target: 75
         flags:
         - purefb
-      Reboot_Required:
+      redis_sentinel:
         target: 75
         flags:
-        - reboot_required
-      Redis_Enterprise:
+        - redis_sentinel
+      redisenterprise:
         target: 75
         flags:
         - redisenterprise
-      Redis_Sentinel:
-        target: 75
-        flags:
-        - redis_sentinel
-      Redpanda:
+      redpanda:
         target: 75
         flags:
         - redpanda
-      Riak_MDC_Replication:
+      riak_repl:
         target: 75
         flags:
         - riak_repl
-      Sendmail:
+      sendmail:
         target: 75
         flags:
         - sendmail
-      Snmpwalk:
+      snmpwalk:
         target: 75
         flags:
         - snmpwalk
-      Sortdb:
+      sortdb:
         target: 75
         flags:
         - sortdb
-      Stardog:
+      speedtest:
+        target: 75
+        flags:
+        - speedtest
+      stardog:
         target: 70
         flags:
         - stardog
-      Storm:
-        target: 75
-        flags:
-        - storm
-      Syncthing:
+      syncthing:
         target: 75
         flags:
         - syncthing
-      TiDB:
+      tidb:
         target: 75
         flags:
         - tidb
-      Traefik:
+      traefik:
         target: 75
         flags:
         - traefik
-      Trino:
+      trino:
         target: 75
         flags:
         - trino
-      UPSC:
-        target: 75
-        flags:
-        - upsc
-      Unbound:
+      unbound:
         target: 75
         flags:
         - unbound
-      Unifi_Console:
+      unifi_console:
         target: 75
         flags:
         - unifi_console
-      Vespa:
+      upsc:
+        target: 75
+        flags:
+        - upsc
+      vespa:
         target: 75
         flags:
         - vespa
-      Zabbix:
+      zabbix:
         target: 50
         flags:
         - zabbix
-      cert-manager:
-        target: 75
-        flags:
-        - cert_manager
-      exim:
-        target: 75
-        flags:
-        - exim
-      cfssl:
-        target: 75
-        flags:
-        - cfssl
-      nvml:
-        target: 75
-        flags:
-        - nvml
-      open_policy_agent:
-        target: 75
-        flags:
-        - open_policy_agent
-      pihole:
-        target: 75
-        flags:
-        - pihole
-      PureFA:
-        target: 75
-        flags:
-        - purefa
-      PureFB:
-        target: 75
-        flags:
-        - purefb
-      speedtest:
-        target: 75
-        flags:
-        - speedtest
```

Comment:
Before I look at the latest changes in more detail, it looks like something's gone wrong with the .codecov.yml. Can we please revert all these changes that shouldn't be here and are creating a conflict?

The diff for this PR should only show changes related to the fiddler integration and leave all others untouched. Please rebase this to current master (as instructed in https://github.com/DataDog/integrations-extras/pull/1383#discussion_r965145550) or merge current master into the branch, and solve the conflicts so that, again, the diff is reduced to changes related to fiddler. If you're still unsure how to do this, I can try to resolve the conflict myself.

Reply:
Also, I tried running the pip within the embedded folder to install the fiddler-client using the command that you provided - /opt/datadog-agent/embedded/bin/pip install fiddler-client==1.4.0. That installation succeeded, however when I restarted the agent, I am still seeing the exact same error. About not being able to import module fiddler:

--------
2022-09-20 18:40:27 UTC | CORE | ERROR | (pkg/collector/scheduler.go:201 in getChecks) | Unable to load a check from instance of config 'fiddler': JMX Check Loader: check is not a jmx check, or unable to determine if it's so; Python Check Loader: unable to import module 'fiddler': No module named 'fiddler'; Core Check Loader: Check fiddler not found in Catalog
2022-09-20 18:40:27 UTC | CORE | ERROR | (pkg/collector/scheduler.go:248 in GetChecksFromConfigs) | Unable to load the check: unable to load any check from config 'fiddler'
2022-09-20 18:40:27 UTC | CORE | INFO | (pkg/collector/scheduler/scheduler.go:92 in Enter) | Scheduling check cpu with an interval of 15s